src/status_im/subs.cljs

```diff
@@ -47,153 +47,144 @@
 
 ;; TOP LEVEL ===========================================================================================================
 
-(def subs-keys (atom {}))
-
-(defn reg-root-key-sub [sub-name db-key keys-key]
-  (swap! subs-keys update keys-key conj db-key)
-  (reg-sub sub-name :<- [keys-key] (fn [db] (get db db-key))))
+(defn reg-root-key-sub [sub-name db-key]
```

Comment:
you can add a doc string that explains why (get db db-key) is safer than the shorthand (reg-sub :bla :bla) which can cause very odd errors which is why reg-root-key-sub is better for root keys that the existing shorthand.